help/en/html/apps/StoringYourWork.shtml

```diff
@@ -1,209 +1,17 @@
-      "backupPanels" in the <a href="../doc/Technical/ProfileFileStructure.shtml">Profile Location</a>) is there.  Best 
-      practice in working with PanelPro is to almost always use <strong>"Store configuration and panels"</strong> or <strong>"Store Panels..."</strong>
-      where it appears, and to rename the file you are saving so you have multiple versions named by you to go back to if necessary.</p>
- 
-      </div>
-
- 
-<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
-      <a name="autosave" id="autosave"></a>
-      <h2>Things that are stored automatically</h2>
-      <div class="para">      
-
-
-      <ul>
-       <li>Window positions and sizes (not all are saved)</li>
-       <li><a href="DecoderPro/index.shtml">DecoderPro</a>
-        <ul>
-         <li><a href="DecoderPro/Roster.shtml"><strong>Roster entry</strong></a>: Automatically stored when changed on the main roster page</li>
-         <li><strong>Roster Group</strong>: new, including changes to Roster Group Table Association; also renaming and deletion of groups</li>
-        </ul>
-       </li>
-       <li><a href="../../package/jmri/jmrit/operations/Operations.shtml">OperationsPro</a> data: automatically stored when data is changed.</li>
-      </ul>
-      <br>
-
-  <p><a href="#TopOfPage">[Go to top of page]</a></p>     
-
-      </div>
-
-
-<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
-     <a name="nosave" id="nosave"></a>
-     <h2>Things that are NOT stored after you quit JMRI</h2>
-      <div class="para">      
- 
-      <p>Some data you will create during your JMRI session or that will be created via communication with your layout is <em>NOT</em> saved
-      after you quite JMRI.  This is normal as this data represents the current state of your trains and layout (e.g. turnout position) so
-      requires feedback from you or your layout.  These transitory data are:</p>
-      <ul>
-       <li>Sensor state</li>
-       <li>Turnout state</li>
-       <li>DispatcherPro Trains</li>
-       <li>[Possibly incomplete list]</li>
-      </ul>
-      <br>
-
-  <p><a href="#TopOfPage">[Go to top of page]</a></p>     
-      </div>
-
-<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
- 
- 
- <!--   <p><a href="#TopOfPage">[Go to top of page]</a></p>     -->
-
-
-   <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
-   
-      <!--#include virtual="/Footer.shtml" -->
-    </div><!-- closes #mainContent-->
-  </div><!-- closes #mBody-->
+<p>This page has been moved to another location
+(<a href="LoadStoreWork">LoadStoreWork.shtml</a>)
+as part of updating our web site.</p>
+<p>Please update your bookmarks.</p>
+<p>If you got to this page from another JMRI page, please let us know so we can fix it.</p>
```

Comment:
Personally, I like to leave this kind of page up for 6 seconds so that the person has a chance of actually reading it - and letting us know if there is some old link we need to change.

Reply:
Done.  Now has 6 second delay.